src/main/webapp/placeGuides/PlaceGuideManager.js

```diff
@@ -6,18 +6,18 @@
  * classes which are responsible for handling the events.
  */
 class PlaceGuideManager {
-  constructor(placeGuideRepository) {
+  constructor(placeGuideRepository, placeGuideDisplayType, hasSubtitle) {
```

Comment:
Let's not add fields to the constructor that we will need to remove anyway. They cause unnecessary conflicts and dependencies.
You can use some dummy values for the title and the subtitle, or you can merge my branch into yours to get the updated version of PlaceGuideManager.

Reply:
merged your branch with mine.